_posts/2018-08-20-grpc_on_http2.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: gRPC on HTTP/2: Engineering a Robust, High Performance Protocol
+published: true
+permalink: blog/grpc_on_http2
```

Comment:
fwiw, the other blog entries have been using `-` as the separator.  I think it reads better than underscore.   Consider changing the slug.

Reply:
Done